cuba/src/main/java/com/haulmont/cuba/web/app/scheduled/ScheduledTaskEditor.java

```diff
@@ -220,8 +221,13 @@ public void init(Map<String, Object> params) {
 
     private void initUserNameField() {
         userNameField.setOptionStyleProvider(UserDetails::getUsername);
-        userNameField.addValueChangeListener(e -> taskDs.getItem().setUserName(e.getValue().getUsername()));
+        userNameField.addValueChangeListener(e -> taskDs.getItem().setUserName(e.getValue() == null ? null : e.getValue().getUsername()));
         userNameField.setSearchExecutor((searchString, searchParams) -> new ArrayList<>(userRepository.getByUsernameLike(searchString)));
+        userNameField.setEnterPressHandler(enterPressEvent -> {
```

Comment:
In CUBA `userNameField` is a SuggestionPickerFIeld, but here it's a SuggestionField.
I think we should use `SuggestionPickerField` like in CUBA and add `Clear` action to it. For instance:
```
<suggestionPickerField id="userNameField"
                       metaClass="sec$User"
                       minSearchStringLength="1"
                       width="${fieldWidth}">
    <actions>
        <action id="clear" type="picker_clear"/>
        <action id="lookup" type="picker_lookup"/>
        <action id="open" type="picker_open"/>
    </actions>
</suggestionPickerField>
```
Thus there is no need to add `setEnterPressHandler`. Try to replace field and check that it does not break anything.

Reply:
It is possible to replace `suggestionField` with `suggestionPickerField` and add `Clear` action for it. 
Because `SuggestionPickerField` bounded with `io.jmix.core.Entity` class there need to add some casts additionally.

Reply:
I get an exception while opening screen here:
```
userNameField.setOptionStyleProvider(UserDetails::getUsername);
```
```
java.lang.invoke.LambdaConversionException: Invalid receiver type interface io.jmix.core.Entity; not a subtype of implementation type interface org.springframework.security.core.userdetails.UserDetails
	at java.lang.invoke.AbstractValidatingLambdaMetafactory.validateMetafactoryArgs(AbstractValidatingLambdaMetafactory.java:233)
	at java.lang.invoke.LambdaMetafactory.metafactory(LambdaMetafactory.java:303)
	at java.lang.invoke.CallSite.makeSite(CallSite.java:302)
	at java.lang.invoke.MethodHandleNatives.linkCallSiteImpl(MethodHandleNatives.java:307)
	at java.lang.invoke.MethodHandleNatives.linkCallSite(MethodHandleNatives.java:297)
	at com.haulmont.cuba.web.app.scheduled.ScheduledTaskEditor.initUserNameField(ScheduledTaskEditor.java:223)
	at com.haulmont.cuba.web.app.scheduled.ScheduledTaskEditor.init(ScheduledTaskEditor.java:209)
```
Using `userNameField.setOptionStyleProvider(item -> item.getUsername());` resolves the problem.

Reply:
I'm able to reproduce it with Java8, not reproduced with java11.
Usage of method reference has been replaced with lamda